api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -410,6 +418,18 @@ public void unvoidVisit_shouldUnvoidEncountersVoidedWithVisit() {
 		encountersByVisit = Context.getEncounterService().getEncountersByVisit(visit, false);
 		assertEquals(1, encountersByVisit.size());
 	}
+
+	/**
+	 * @see VisitService#getActiveVisitsByPatient(Patient) 
+	 */
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyCollection() {
+		assertEquals(visitService.getVisitsByPatient(null), Collections.emptyList());
```

Comment:
please remove this test as this is already tested in getVisitsByPatient_shouldReturnEmptyListGivenNull